gtk4/sys/src/lib.rs

```diff
@@ -8522,6 +8585,7 @@ extern "C" {
     //=========================================================================
     // GtkExpressionWatch
     //=========================================================================
+    pub fn gtk_expression_watch_get_type() -> GType;
```

Comment:
Missing `Since`

Reply:
I don't think there's a way to mark the `get_type` function with a Since annotation, the type existed before. It's already guarded by the `GDK_AVAILABLE_IN_4_2` macro, but it doesn't seem to help, see https://gitlab.gnome.org/GNOME/gtk/-/blob/master/gtk/gtkexpression.h#L74

Reply:
Yes, that can only be marked as such in the `Gir.toml`. There's no way to annotate it accordingly otherwise.

This is only needed if the `get_type()` function is newer than the type itself, like here.